src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -83,6 +83,8 @@ You can install the version of cwltool that we've tested for use with Dockstore
 pip install setuptools==36.5.0
 pip install cwl-runner cwltool==${this.cwltoolVersion} schema-salad==2.6.20170806163416 avro==1.8.1 ruamel.yaml==0.14.12 requests==2.18.4
 \`\`\`
+If using Python 3, install cwl-avro==1.8.3 instead of avro==1.8.1.
```

Comment:
This is ugly, but maybe a comment for us with the ticket to check to see when this gets fixed. 
Also 1.8.1 is repeated from above, can be a variable like cwltoolVersion